### PR TITLE
Title image preprocessor

### LINKED ISSUE #85: Add the intro icon/number and images to content
- [x] Intro Icons (Light and dark are in)
- [x] Intro Numbers (Remember we have our tailwind in now)

Image

### PR DESCRIPTION
**Pull request by saraswatpuneet**
Problem
=======
Create a title image/number preprocessors for setting title icon and number

Closes #85 

Solution
========
- [x] like svg-embed create a title-image preprocessor
- [x] it should take atleast a number and image as input and display them as icons 

Screenshots (optional):
-----------------------
image

image

### CODE REVIEW

**Comment by saraswatpuneet** [comment]
image

File: pages/Guides/BecomeAProvider.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
+{{#title-image .01 pages/images/icons/lock.svg "Become a Provider"}}
+
```
Comment (saraswatpuneet) [inline pages/Guides/BecomeAProvider.md:2]: currently the pre-processor is not using the third argument but maybe if some better layout comes along, but 1 and 2 arguments are rendered

**Review by wilwade — CHANGES_REQUESTED** [review]
Only see two things so far. Looking good

File: css/highlight.css [hunk 2]
```diff
@@ -175,3 +175,29 @@ pre .hljs {
 .svg-embed-coal {
   display: none;
 }
+
+
+.title-embed {
```
Comment (wilwade) [inline css/highlight.css:180]: Likely need some top and bottom margin on this as well.
image

File: css/highlight.css [hunk 3]
```diff
@@ -175,3 +175,29 @@ pre .hljs {
...
+  display: none;
+  width: 50px;
+  height: 50px;
+  margin-right: 8px;
+}
+
+.embed-number {
+  font-size: 25px;
+  font-weight: bold;
+  color: orange;
```
[the remark this answers is not in the document]
Comment (JoeCap08055) [reply css/highlight.css:202]: Just wondering if this will be changing again, as it seems like the .xyz site has removed the orange in favor of blues...

**Comment by wilwade** [comment]
Perhaps, but this is what is getting approved, so it's what we got

File: css/highlight.css [hunk 4]
```diff
@@ -175,3 +175,29 @@ pre .hljs {
...
+
+.title-embed-light {
+  display: flex;
+  width: 50px;
+  height: 50px;
+  margin-right: 8px;
+}
+
+.title-embed-coal {
+  display: none;
```
[the remark this answers is not in the document]
Comment (saraswatpuneet) [reply css/highlight.css:193]: image

File: css/highlight.css [hunk 5]
```diff
@@ -175,3 +175,29 @@ pre .hljs {
...
+  width: 50px;
+  height: 50px;
+  margin-right: 8px;
+}
+
+.embed-number {
+  font-size: 25px;
```
Comment (wilwade) [inline css/highlight.css:200]: Looks like it should be larger from the design
```suggestion
  font-size: 44px;
```

File: pages/Guides/BecomeAProvider.md [hunk 6]
```diff
@@ -1,3 +1,5 @@
+{{#title-image .01 pages/images/icons/lock.svg "Become a Provider"}}
```
Comment (wilwade) [inline pages/Guides/BecomeAProvider.md:1]: Reminder that we need to add this in for 2-4 well. Or perhaps that is for a separate PR?

**Review by wilwade — COMMENTED** [review]
Only two things left that I can see

File: css/highlight.css (hunk 2 above)
Comment (saraswatpuneet) [reply css/highlight.css:180]: added some margins

image

File: pages/Guides/BecomeAProvider.md (hunk 6 above)
Comment (saraswatpuneet) [reply pages/Guides/BecomeAProvider.md:1]: I can add that in this PR as well

**Review by JoeCap08055 — COMMENTED** [review]
lgtm strictly as far as the preprocessor goes, but I'll defer to @wilwade since he seems to be drilling down into the layout details.

File: pages/Guides/BecomeAProvider.md (hunk 6 above)
Comment (saraswatpuneet) [reply pages/Guides/BecomeAProvider.md:1]: added 2,3 and 4 title images

**Review by wilwade — APPROVED** [review]
All looks good to me. Shipit!

**Pull request merged**